### PR TITLE
Fix coercing false non-null booleans

### LINKED ISSUE #279: Cannot pass false as the value to a "Boolean!" argument
### What did you do?

Tried a query like this:

```graphql
mutation changeCustomerNumber($customerId: ID!, $customerNumber: String!, $postalCode: String!, $isPersonal: Boolean!) {
  changeCustomerNumber(customerId: $customerId, customerNumber: $customerNumber, postalCode: $postalCode, isPersonal: $isPersonal)
}
```

With these variables:

```json
{
  "customerId": "blabla",
  "customerNumber": "123",
  "isPersonal": false,
  "postalCode": "12345"
}
```
### What did you expect to happen?

Everything to work

### What actually happened?

```json
{
  "errors": [
    {
      "message": "Variable \"$isPersonal\" got invalid value false; Expected non-nullable type Boolean! not to be null.",
      "locations": [
        {
          "line": 1,
          "column": 97
        }
      ],
      "path": []
    }
  ],
  "data": null
}
```

### PR DESCRIPTION
**Pull request by Jalle19**
Fixes #279

### CODE REVIEW

File: src/Execution/ValuesHelper.php [hunk 1]
```diff
@@ -293,7 +293,7 @@ protected function coerceValueForNonNullType(
         NodeInterface $blameNode,
         ?Path $path
     ): CoercedValue {
-        if (empty($value)) {
+        if ($value === null) {
```
Comment (crisu83) [inline src/Execution/ValuesHelper.php]: @Jalle19 Can you flip this comparison for consistency?

File: tests/Functional/Execution/ValuesHelperTest.php [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+                                'name' => [
+                                    'type' => stringType(),
+                                ]
+                            ]
+                        ]
+                    ]
+                ])
+        ]);
+
+
```
Comment (crisu83) [inline tests/Functional/Execution/ValuesHelperTest.php]: Also here's an extra line-break.

File: tests/Functional/Execution/ValuesHelperTest.php [hunk 3]
```diff
@@ -0,0 +1,102 @@
...
+     * @throws \Digia\GraphQL\Error\InvariantException
+     * @throws \Digia\GraphQL\Error\SyntaxErrorException
+     */
+    public function testCoerceVariableValues(): void
+    {
+        // Test that non-nullable booleans are handled correctly
+        $schema = newSchema([
+            'query' =>
+                newObjectType([
+                    'name'   => 'NonNullBoolean',
```
Comment (crisu83) [inline tests/Functional/Execution/ValuesHelperTest.php]: This type name should start with a lower-case letter.

File: tests/Functional/Execution/ValuesHelperTest.php [hunk 4]
```diff
@@ -0,0 +1,106 @@
...
+                                'name' => [
+                                    'type' => stringType(),
+                                ]
+                            ]
+                        ]
+                    ]
+                ])
+        ]);
+
+        $documentNode = parse('query Hello($name: String) { Greeting(name: $name) }');
```
Comment (crisu83) [inline tests/Functional/Execution/ValuesHelperTest.php]: Ah I missed this query last time, I would use line-breaks here too.

Comment (Jalle19) [reply tests/Functional/Execution/ValuesHelperTest.php]: Like earlier, this was already like this, GitHub's diff just doesn't understand that this is existing code

**Pull request merged**